src/app/facilities/facilities.component.ts

```diff
@@ -0,0 +1,36 @@
+import {AfterViewInit, Component, OnInit} from '@angular/core';
+import {FacilitiesDescriptions} from "./utilities/faciilities-descriptions";
+import {animate, animation, style, transition, trigger} from "@angular/animations";
+import {fadeSlideInOutAnimation} from "../shared/animations";
+
+@Component({
+  selector: 'app-facilities',
+  templateUrl: './facilities.component.html',
+  styleUrls: ['./facilities.component.scss'],
+  animations: [fadeSlideInOutAnimation]
+})
+export class FacilitiesComponent implements OnInit{
+    public descriptionToDisplay?: string;
```

Comment:
Nie można temu od razu przypisać wartości first, skoro potem i tak to robimy w ngOnInit?